zstackctl/zstackctl/utils/concurrentlog_handler.py

```
@@ -0,0 +1,337 @@
+# class, right? We do some rename checking... that should prevent some file
+# clobbering that the builtin class allows.
+
+# sibling module than handles all the ugly platform-specific details of file locking
+from portalocker import lock, unlock, LOCK_EX, LOCK_NB, LockException
+
+
+# Workaround for handleError() in Python 2.7+ where record is written to stderr
+class NullLogRecord(LogRecord):
+
+    def __init__(self):
+        super(NullLogRecord, self).__init__()
```

Pylint (3.3.7)

[error] No value for argument 'name' in method call

(E1120)